tracing/src/main/java/org/axonframework/extensions/tracing/SpanUtils.java

```diff
@@ -41,24 +41,28 @@
      * <p>
      * This method will check if the message is a
      * <ul>
-     * <li>{@link QueryMessage}, returning "{@code QueryMessage}",</li>
-     * <li>{@link CommandMessage}, returning "{@code CommandMessage}",</li>
-     * <li>{@link EventMessage}, returning "{@code EventMessage}",</li>
+     * <li>{@link QueryMessage}, returning  the queryName",</li>
+     * <li>{@link CommandMessage}, returning the commandName",</li>
+     * <li>{@link EventMessage}, returning the event type,</li>
      * <li>otherwise returns "{@code Message}"</li>
```

Comment:
This "otherwise" scenario is the same as the `EventMessage` scenario at the moment. The javadoc should make this clear.